pychunkedgraph/io/edge_storage.py

```diff
@@ -0,0 +1,160 @@
+"""
+Functions for reading and writing edges 
+to secondary storage with CloudVolume
+"""
+
+import os
+from typing import List, Dict, Tuple, Union
+
+import numpy as np
+import zstandard as zstd
+
+from cloudvolume import Storage
+from cloudvolume.storage import SimpleStorage
+from .protobuf.chunkEdges_pb2 import ChunkEdges
+
+
+def _decompress_edges(content: bytes) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    """
+    :param content: zstd compressed bytes
+    :type bytes:
+    :return: edges, affinities, areas
+    :rtype: Tuple[np.ndarray, np.ndarray, np.ndarray]
+    """
+
+    def _get_edges(
+        edge_type: str, edgesMessage: Union[ChunkEdges, ChunkEdges.Edges]
+    ) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+        if edge_type == "cross":
```

Comment:
Would like to store these strings as constants/enum or whatever the typical Python way is to avoid spelling mistakes / make it easier to change the names in the future.